cmd/manta-init.js

```diff
@@ -631,6 +635,20 @@ var pipelineFuncs = [
 		});
 	},
 
+	function checkMantaApplicationVersion(_, cb) {
+		if (self.manta_app &&
+		    self.manta_app.metadata['MANTAV'] !== MANTA_VERSION) {
+			return cb(new VError(
+			    'A v%s Manta application was found on this ' +
+			    'Triton instance which conflicts with the ' +
+			    'version being deployed. ' +
+			    'You must delete that application before ' +
```

Comment:
Or perhaps somehow resolve the situation it rather than just go delete it?  If we hit this situation, then *something* is wrong/fishy, but I'm not sure that we know the right answer is for the operator to start deleting SAPI applications, do we?

Reply:
What could possibly go wrong!? You're absolutely right though, I'll say 'resolve the problem' instead. Hopefully if we get to writing a tool that manages the upgrade path, we'll be able to update this text to mention that at some point.

Reply:
```
[root@a033611c-1f3e-4d1a-96fe-a59823ebe99d (coal-1:manta0) /opt/smartdc/manta-deployment]# manta-init -e tim.foster@joyent.com -s coal
logs at /var/log/manta-init.log
Error: A v2 Manta application was found on this Triton instance which conflicts with the version being deployed. You must resolve that problem before attempting to deploy a new Manta application.
[root@a033611c-1f3e-4d1a-96fe-a59823ebe99d (coal-1:manta0) /opt/smartdc/manta-deployment]#
```